tf2_eigen/CMakeLists.txt

```diff
@@ -25,5 +25,6 @@ if(CATKIN_ENABLE_TESTING)
 
   catkin_add_gtest(tf2_eigen-test test/tf2_eigen-test.cpp)
   target_link_libraries(tf2_eigen-test ${catkin_LIBRARIES} ${GTEST_LIBRARIES})
+  catkin_add_nosetests(test/test_tf2_sensor_msgs.py)
```

Comment:
once done, try building everything again and run "make run_tests" or whatever you want from here: http://docs.ros.org/indigo/api/catkin/html/howto/format1/run_tests.html